libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -401,6 +404,17 @@ class YulUtilFunctions
 
 	std::string readFromMemoryOrCalldata(Type const& _type, bool _fromCalldata);
 
+	/// @returns a function that reads a value type from storage.
+	/// Performs bit mask/sign extend cleanup and appropriate left / right shift, but not validation.
+	/// @param _splitFunctionTypes if false, returns the address and function signature in a
+	/// single variable.
+	std::string readFromStorageValueType(Type const& _type, size_t _offset, bool _splitFunctionTypes);
+	std::string readFromStorageValueTypeDynamic(Type const& _type, bool _splitFunctionTypes);
+
+	/// @returns a function that reads a reference type from storage.
+	/// if target type is in memory it allocates memory needed to copy actual type
```

Comment:
```suggestion
	/// @returns a function that reads a reference type from storage to memory (performing a deep copy).
```
If I understood this correctly :-)?

Reply:
(or put differently: what "target type" are you talking about :-)?)

Reply:
You are right, will update like suggested. :-)